macros/upload_artifacts_v2.sql

```diff
@@ -84,7 +156,56 @@
 
         )
 
-        {{ flatten_manifest("raw_data") }};
+        {{ flatten_manifest("raw_data") }}
+
+    ) as new_data
+    on old_data.artifact_run_id = new_data.artifact_run_id and old_data.artifact_generated_at = new_data.artifact_generated_at
```

Comment:
```suggestion
    on old_data.artifact_run_id = new_data.artifact_run_id and old_data.node_id = new_data.node_id
```

Reply:
In line with #103 and #94 - we should use `command_invocation_id` (+/- `node_id`) for these merges.